spec/requests/admin/users_manage_spec.rb

```diff
@@ -164,6 +164,51 @@ def full_profile
       put "/admin/users/#{user.id}", params: { user: { remove_credits: "3" } }
       expect(user.credits.size).to eq(2)
     end
+
+    it "removes non-admin roles from non-super_admin users", :aggregate_failures do
+      user.add_role(:trusted)
+
+      expect do
+        delete "/admin/users/#{user.id}", params: { user_id: user.id, role: :trusted }
+      end.to change(user.roles, :count).by(-1)
+
+      expect(user.has_role?(:trusted)).to be false
+      expect(request.flash["success"]).to include("successfully removed from the user!")
+    end
+
+    it "removes the correct resource_admin_role from non-super_admin users", :aggregate_failures do
```

Comment:
Should we update this spec and make sure the user has two single resource rules first and that only one of them gets removed?

Reply:
Great suggestion - done! ✔️